package.json

```diff
@@ -1,36 +1,30 @@
 {
   "name": "instantsearch.css",
+  "type": "module",
   "version": "7.4.5",
   "description": "instantsearch.css is a CSS library created to help all InstantSearch users easily customize their widgets",
   "author": "Algolia <support@algolia.com>",
   "license": "MIT",
   "repository": "https://github.com/algolia/instantsearch-specs",
   "scripts": {
-    "build": "./scripts/build.sh",
-    "dev": "gulp & hugo server & wait",
-    "prepublishOnly": "npm run build"
+    "dev": "yarn styles:app && astro dev",
+    "start": "yarn styles:app && astro dev",
+    "check": "astro check && tsc",
+    "build": "yarn styles:app && astro build",
+    "preview": "astro preview",
+    "astro": "astro",
+    "styles:app": "sass src/styles/themes:public/styles --style compressed",
+    "styles:package": "sass src/styles/themes:themes --no-source-map",
+    "styles:min": "sass src/styles/themes/algolia.scss:themes/algolia-min.css src/styles/themes/satellite.scss:themes/satellite-min.css src/styles/themes/reset.scss:themes/reset-min.css --no-source-map --style compressed",
+    "styles:autoprefixer": "postcss themes/ --use autoprefixer --replace --no-map",
```

Comment:
because of this it would be a breaking change, I'm ok with that (as we've had plenty major versions of this package), just need to make sure to update the docs